via/templates/pdf_viewer.html.jinja2

```diff
@@ -26,20 +26,9 @@
 
     {# Configure Hypothesis client. #}
     <script>
-      window.hypothesisConfig = function() {
-        return {
-          showHighlights: true,
-          appType: 'via',
-
-          {% if h_request_config %}
-          requestConfigFromFrame: '{{ h_request_config}}',
-          {% endif %}
-
-          {% if h_open_sidebar %}
-          openSidebar: true,
-          {% endif %}
-        };
-      }
+        window.hypothesisConfig = function() {
+            return {{ hypothesis_config | tojson }};
+        }
```

Comment:
This is a change we know we are going to have to make. Rather than write the tests twice I thought I'd just get it done.